src/main/java/net/mcreator/workspace/references/ElementReference.java

```diff
@@ -27,6 +27,7 @@
 @Target(ElementType.FIELD) @Retention(RetentionPolicy.RUNTIME) public @interface ElementReference {
 
 	String[] defaultValues() default "";
+
 	String customPrefix() default "";

```

Comment:
Is this field needed at all? In other words, may there be any `customPrefix`es except `CUSTOM:`?

Reply:
I guess not